File: archive/final-value-of-variable-after-performing-operations.js
const finalValueAfterOperations = function(operations) {
  let sum = 0
  operations.forEach(el => {
    if (el === '--X' || el === 'X--') {
      sum--
    } else {
      sum++
    }
  })

  return sum
};

console.log(finalValueAfterOperations(["--X","X++","X++"]))

File: final-value-of-variable-after-performing-operations.go
package main

import "fmt"

func main() {
	fmt.Println(finalValueAfterOperations([]string{"--X", "X++", "X--"}))
}

func finalValueAfterOperations(operations []string) int {
	x := 0

	for i := 0; i < len(operations); i++ {
		if operations[i] == "--X" {
			x--
		}
		if operations[i] == "X--" {
			x--
		}
		if operations[i] == "++X" {
			x++
		}
		if operations[i] == "X++" {
			x++
		}
	}

	return x
}
